<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>708</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Imgs/gear 3.png</normaloff>Imgs/gear 3.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
  background-color: rgb(105, 171, 216);

</string>
  </property>
  <property name="iconSize">
   <size>
    <width>25</width>
    <height>25</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true">QLabel{
font: 12pt &quot;MS Shell Dlg 2&quot;;
	font: 12pt &quot;Century Gothic&quot;;
background: transparent;
}
QGroupBox{
background-color: rgba(250, 250, 250, 250);
border-radius: 5px;
}
QSpinBox{
background-color: rgb(200, 200, 200);
	font: 12pt &quot;MS Shell Dlg 2&quot;;
}

QDoubleSpinBox{
background-color: rgb(200, 200, 200);
	font: 12pt &quot;MS Shell Dlg 2&quot;;
}
QComboBox{
background-color: rgb(200, 200, 200);
	font: 75 12pt &quot;MS Shell Dlg 2&quot;;
}
Line{
color: rgb(175, 175, 175);
}
QPushButton{
background-color: rgb(200, 200, 200);
	font: 75 12pt &quot;MS Shell Dlg 2&quot;;
}
QPushButton:hover{
background-color: rgb(225, 225, 225);

}
</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="8" column="1">
        <widget class="QDoubleSpinBox" name="Phrase_doubleSpinBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Non speaking duration _:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QDoubleSpinBox" name="Pause_doubleSpinBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="13" column="0" colspan="3">
        <widget class="Line" name="line_4">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>2</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="3">
        <widget class="Line" name="line_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>2</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Pause Threshold ______:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="3">
        <widget class="Line" name="line_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>2</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="3">
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: rgba(0, 0, 0, 210);</string>
         </property>
         <property name="text">
          <string>[seconds of non-speaking audio
 before a phrase is considered complete]</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="Line" name="line">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>2</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QComboBox" name="comboBox"/>
       </item>
       <item row="14" column="2">
        <widget class="QPushButton" name="Reset_operation_timeout_Button">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="Reset_Energy_Button">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>28</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: rgba(0, 0, 0, 210);</string>
         </property>
         <property name="text">
          <string>[minimum audio energy to consider for recording]</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QPushButton" name="Reset_Pause_Button">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QPushButton" name="Reset_Phrase_Button">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item row="11" column="2">
        <widget class="QPushButton" name="Reset_non_speaking_duration_Button">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0" colspan="3">
        <widget class="QLabel" name="label_8">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: rgba(0, 0, 0, 210);</string>
         </property>
         <property name="text">
          <string>[seconds of non-speaking audio
 to keep on both sides of the recording]</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="3">
        <widget class="QLabel" name="label_7">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: rgba(0, 0, 0, 210);</string>
         </property>
         <property name="text">
          <string>[minimum seconds of speaking audio
 before we consider the speaking audio a phrase
 - values below this are ignored
 (for filtering out clicks and pops) ]</string>
         </property>
        </widget>
       </item>
       <item row="15" column="0" colspan="3">
        <widget class="QLabel" name="label_10">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: rgba(0, 0, 0, 210);</string>
         </property>
         <property name="text">
          <string>[seconds after an internal operation starts
 before it times out, or 'None'  for no timeout]</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Operation Timeout ____:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Phrase Threshold _____:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="Energy_spinBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Energy Threshold _____:</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QDoubleSpinBox" name="Non_speaking_daration_doubleSpinBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="Line" name="line_11">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>2</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_11">
         <property name="styleSheet">
          <string notr="true">border-color: 1px solid rgb(0, 0, 127);</string>
         </property>
         <property name="text">
          <string>Audio Input Devices:</string>
         </property>
        </widget>
       </item>
       <item row="14" column="1">
        <widget class="QDoubleSpinBox" name="TimeOutDoubleSpinBox"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="Reset_pushButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 padding-top:2px;
	font: 14pt &quot;MS Shell Dlg 2l&quot;;
 color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(33, 33, 33, 255), stop:1 rgba(75, 75, 75, 255));
background-color:qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(0, 244, 255, 255), stop:1 rgba(3, 115, 255, 255));

border:1px solid rgb(253, 254, 254);

border-radius:5px;
}
QPushButton:hover{
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(63, 63, 63, 255), stop:1 rgba(33, 33, 33, 255));
 background-color: qlineargradient(spread:pad, x1:0.505636, y1:0.221, x2:0.431818, y2:1, stop:0 rgba(89, 255, 255, 255), stop:1 rgba(60, 138, 255, 255));

}

QPushButton:pressed{
background-color: qlineargradient(spread:pad, x1:0.767, y1:1, x2:1, y2:0, stop:0 rgba(0, 244, 255, 255), stop:1 rgba(3, 115, 255, 255));\n&quot;

}</string>
      </property>
      <property name="text">
       <string>Reset All</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="Line" name="line_8">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="Line" name="line_9">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="Line" name="line_10">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="Line" name="line_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="Developer_recomendation_Button">
      <property name="styleSheet">
       <string notr="true">QPushButton{
 padding-top:2px;
	font: 14pt &quot;MS Shell Dlg 2l&quot;;
 color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(33, 33, 33, 255), stop:1 rgba(75, 75, 75, 255));
background-color:qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(0, 244, 255, 255), stop:1 rgba(3, 115, 255, 255));

border:1px solid rgb(253, 254, 254);

border-radius:5px;
}
QPushButton:hover{
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(63, 63, 63, 255), stop:1 rgba(33, 33, 33, 255));
 background-color: qlineargradient(spread:pad, x1:0.505636, y1:0.221, x2:0.431818, y2:1, stop:0 rgba(89, 255, 255, 255), stop:1 rgba(60, 138, 255, 255));

}

QPushButton:pressed{
background-color: qlineargradient(spread:pad, x1:0.767, y1:1, x2:1, y2:0, stop:0 rgba(0, 244, 255, 255), stop:1 rgba(3, 115, 255, 255));\n&quot;

}</string>
      </property>
      <property name="text">
       <string>Set as  Recommendated</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="Line" name="line_7">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="2">
     <widget class="Line" name="line_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="Apply_pushButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 padding-top:2px;
	font: 15pt &quot;MS Shell Dlg 2l&quot;;
 color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(33, 33, 33, 255), stop:1 rgba(75, 75, 75, 255));
background-color:qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(0, 244, 255, 255), stop:1 rgba(3, 115, 255, 255));

border:1px solid rgb(253, 254, 254);

border-radius:5px;
}
QPushButton:hover{
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(63, 63, 63, 255), stop:1 rgba(33, 33, 33, 255));
 background-color: qlineargradient(spread:pad, x1:0.505636, y1:0.221, x2:0.431818, y2:1, stop:0 rgba(89, 255, 255, 255), stop:1 rgba(60, 138, 255, 255));

}

QPushButton:pressed{
background-color: qlineargradient(spread:pad, x1:0.767, y1:1, x2:1, y2:0, stop:0 rgba(0, 244, 255, 255), stop:1 rgba(3, 115, 255, 255));\n&quot;

}</string>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="Manage_pushButton">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 padding-top:2px;
	font: 12pt &quot;MS Shell Dlg 2l&quot;;
 color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(33, 33, 33, 255), stop:1 rgba(75, 75, 75, 255));
	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(200, 200, 200, 255), stop:1 rgba(255, 255, 255, 255));
border:1px solid rgb(253, 254, 254);

border-radius:5px;
}
QPushButton:hover{
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(63, 63, 63, 255), stop:1 rgba(33, 33, 33, 255));
 background-color: qlineargradient(spread:pad, x1:0.505636, y1:0.221, x2:0.431818, y2:1, stop:0 rgba(89, 255, 255, 255), stop:1 rgba(60, 138, 255, 255));

}

QPushButton:pressed{
background-color: qlineargradient(spread:pad, x1:0.767, y1:1, x2:1, y2:0, stop:0 rgba(0, 244, 255, 255), stop:1 rgba(3, 115, 255, 255));\n&quot;

}</string>
      </property>
      <property name="text">
       <string>Manage recording devices</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
